pkg/cloud/gke/storage/bucket_provider.go

```diff
@@ -74,12 +79,15 @@ func (b *GKEBucketProvider) EnsureBucketIsCreated(bucketURL string) error {
 }
 
 // UploadFileToBucket uploads a file to the provided GCS bucket with the provided outputName
-func (b *GKEBucketProvider) UploadFileToBucket(reader io.Reader, outputName string, bucketURL string) (string, error) {
-	bytes, err := ioutil.ReadAll(reader)
+func (b *GKEBucketProvider) UploadFileToBucket(reader io.Reader, key string, bucketURL string) (string, error) {
+	data, err := ioutil.ReadAll(reader)
 	if err != nil {
 		return "", err
 	}
-	return gke.UploadFileToBucket(bytes, outputName, bucketURL)
+
+	log.Logger().Debugf("Uploading data %s to bucket %s with key %s", string(data), bucketURL, key)
```

Comment:
I tweaked this in #6252 to not include the data in the debug log, since, well, that's gonna be a a lot of text. Probably worth doing here too - I don't think there's really any value in seeing the data we're about to write, and I have a feeling that trying to log that much at once may be causing this log line not to show up at all.

Reply:
ooh lemme do the same! :) was a copy/paste

Reply:
Yup. =) Thanks!